corehq/apps/accounting/utils/invoicing.py

```diff
@@ -59,7 +60,7 @@ def _get_domains_over_threshold(invoices, today, get_oldest_invoice_fn):
 
 
 def get_accounts_with_customer_invoices_overdue(today):
```

Comment:
Want to confirm my understanding of the operation here. All accounts returned by this function won't be downgraded yet, right?

Reply:
That's right. There's still the check to make sure we don't automatically downgrade Customer Billing Accounts, but they will get the invoice reminder, the overdue notice, and the downgrade _warning_ emails in the normal timelines.

https://github.com/dimagi/commcare-hq/blob/fbef7976c6c4b6af71e00d6fcbe27454b8346f82/corehq/apps/accounting/utils/unpaid_invoice.py#L207-L210